app/scripts/filters/resources.js

```diff
@@ -125,6 +125,50 @@ angular.module('openshiftConsole')
       return displayName;
     };
   })
+  .filter('canI', function(AuthorizationService) {
+    return function(resource, verb) {
+      return AuthorizationService.canI(resource, verb);
+    };
+  })
+  .filter('canIDoAny', function(canIFilter) {
+    var resourceRulesMap = {
```

Comment:
We can't hardcode this list, especially for the other resources page because that list is completely dynamic and we will never know all of the 3rd party extension types that may show up.   This filter needs to take in both a resource, and a list of verbs it should check.